src/unix/fs.c

```diff
@@ -202,7 +202,11 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
   ts[0].tv_nsec = (uint64_t)(req->atime * 1000000) % 1000000 * 1000;
   ts[1].tv_sec  = req->mtime;
   ts[1].tv_nsec = (uint64_t)(req->mtime * 1000000) % 1000000 * 1000;
+#if !(defined(__ANDROID_API__) && __ANDROID_API__ < 21)
```

Comment:
Style: just `#if defined(__ANDROID_API__) &&__ANDROID_API__ < 21` and swap the clauses.

Substance: it's possible that the kernel supports the system call even if the libc symbol is missing. It would be better to make the system call directly (see `src/unix/linux-syscalls.[ch]`) and fall back to `utimensat()` when `futimens()` fails with `ENOSYS`. That way you don't need platform-specific compile guards either.

(Not a blocker but a recommendation.)